index.html

```diff
@@ -1492,8 +1494,7 @@ <h3>
           </li>
           <li>Set <var>manifest</var>["<a>start_url</a>"] to the result of
           running <a>processing the <code>start_url</code> member</a> given
-          <var>manifest</var>["<a>start_url</a>"], <var>manifest URL</var>, and
-          <var>document URL</var>.
+          <var>manifest</var>["<a>start_url</a>"] and <var>manifest URL</var>.
           </li>
           <li>Set <var>manifest</var>["<a>lang</a>"] to the result of running
           <a>processing the <code>lang</code> member</a> given
```

Comment:
This comment is for line 1506. start_url can now be undefined, the processing for scope segfaults.

Reply:
Specs don't segfault :)

This is WAI. "undefined" here doesn't mean set it to nullptr in C++. It means (according to [how a JS value is converted into a dictionary member](https://heycam.github.io/webidl/#es-dictionary)) to leave it empty as if the key didn't exist at all. So this logic means to treat an invalid `start_url` exactly the same as a missing `start_url`.

(Note that the WebIDL data structure permits only two options for `start_url`: missing or a valid-but-possibly-empty string. So it should not be possible to store some other null-type value in here.)

C++ implementations should make sure this is the case and of course not segfault.